src/redux/actions/projects/updateProject.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash';
+
+import {
+  PROJECTS_UPDATE,
+} from '../../actionTypes/projects';
+
+const createProject = (
```

Comment:
The name here still says `createProject`.

Isn't the `lastModified` value meant to also update here?

Reply:
oh yea :S fixed in commit d78e4f3